comictaggerlib/ui/pagelisteditor.ui

```diff
@@ -108,6 +108,13 @@
          </property>
         </widget>
        </item>
+       <item row="0" column="3">
+        <widget class="QPushButton" name="btnAutoDouble">
+         <property name="text">
+          <string>Auto-Detect Double Pages</string>
+         </property>
+        </widget>
+       </item>
```

Comment:
Putting this button here squishes the UI too much. Put it in the menu below re-calculate page dimensions and in the tool bar is acceptable